### ISSUE TITLE
`VK_KHR_acceleration_structure` device query functions require features not present in the extension

### ISSUE THREAD

**Issue by Rua** [KhronosGroup/Vulkan-Docs#2134]
The [`vkGetAccelerationStructureBuildSizesKHR`](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkGetAccelerationStructureBuildSizesKHR.html) and [`vkGetDeviceAccelerationStructureCompatibilityKHR`](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkGetDeviceAccelerationStructureCompatibilityKHR.html) functions both have a requirement that either the `rayQuery` or the `rayTracingPipeline` features must be enabled. These features are not present in the `VK_KHR_acceleration_structure` extension, but rather in the `VK_KHR_ray_query` and `VK_KHR_ray_tracing_pipeline` extensions (which themselves depend on `VK_KHR_acceleration_structure`, not the other way around).

It would make more sense if these functions required a feature that was actually present in the extension, such as `accelerationStructure`.

**Comment by dgkoch** [comment]
You're right. I'm guessing this must have been left over from when there was only one monolithic extension. 

Will be fixed in a future spec update.
